test/functional/apps/visualize/_area_chart.js

```diff
@@ -153,5 +155,85 @@ export default function ({ getService, getPageObjects }) {
       const sideEditorExists = await PageObjects.visualize.getSideEditorExists();
       expect(sideEditorExists).to.be(false);
     });
+
+    describe('switch between Y axis scale types', () => {
+      before(initAreaChart);
+      const axisId = 'ValueAxis-1';
+
+      it('should show ticks on selecting log scale', async () => {
+        await PageObjects.visualize.clickMetricsAndAxes();
+        await PageObjects.visualize.clickYAxisOptions(axisId);
+        await PageObjects.visualize.selectYAxisScaleType(axisId, 'log');
+        await PageObjects.visualize.clickYAxisAdvancedOptions(axisId);
+        await PageObjects.visualize.changeYAxisFilterLabelsCheckbox(axisId, false);
+        await PageObjects.visualize.clickGo();
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
You can remove the `waitUntilLoadingHasFInished` behind all `clickGo`, since the `clickGo` function should (and does) wait for all rendering that is needed before finishing.